xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-ui/src/main/resources/Refactoring/Code/RefactoringConfiguration.xml

```diff
@@ -36,7 +36,43 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content/>
+  <content>{{velocity}}
+#if ($xwiki.hasRecycleBin())
```

Comment:
No else to handle?

Reply:
Currently this is the only value configurable, when the recycle bin is not activated, their is nothing to display. This is why we are also hiding the entry in the menu.

Reply:
> This is why we are also hiding the entry in the menu.

Ok I missed that part. I would still display a warning message in the else part, in case someone end up there.

Reply:
Alternative message added https://github.com/xwiki/xwiki-platform/pull/1413/files#diff-3c735d937df88dc14ac06a727039ad99c06968b61044fa89899f7fb5564be338R76

Reply:
thanks